.github/copilot-instructions.md

```diff
@@ -73,9 +73,9 @@ curl -s "https://api.github.com/repos/Azure/azure-rest-api-specs/commits?path=<p
 ## EXECUTION SEQUENCE - 7 MANDATORY STEPS
 
 **ESTIMATED TOTAL TIME: 10-15 minutes**
-- SDK Generation: 5-6 minutes
-- Static Validation: 3-5 minutes  
-- Documentation & Commit: 2-4 minutes
+- SDK Generation: ~2 minutes
+- Static Validation: ~3-5 minutes
+- Documentation & Commit: ~2-4 minutes
```

Comment:
Nit: does the agent need to know this? Just wondering how this data is helps the agent?

Reply:
its for the user moreso